cpp/ql/src/semmle/code/cpp/security/TaintTracking.qll

```diff
@@ -1,5 +1,7 @@
 /*
  * Support for tracking tainted data through the program.
+ *
+ * Prefer to use `semmle.code.cpp.dataflow.TaintTracking` when designing new queries.
```

Comment:
@jbj I'd also like to check you agree with this advice, and that I shouldn't be recommending `ir.dataflow.tainttracking` instead?

Reply:
I don't think that library is entirely ready yet, but @rdmarsh2 is working on it. See https://github.com/github/codeql/pull/3320#issuecomment-619211810 and https://github.com/github/codeql/pull/3364.

Reply:
Sorry I'm a bit confused - which taint library / libraries would you currently recommend the developer of a new query use / choose from?

Reply:
I have a bad habit of writing "it" and "that" without making the object of reference clear...

The `semmle.code.cpp.ir.dataflow.TaintTracking` library isn't entirely ready yet, so for now we should keep recommending `semmle.code.cpp.dataflow.TaintTracking`. In other words, this comment line LGTM!

Reply:
Great, thanks!